CondCore/BeamSpotPlugins/interface/BeamSpotPayloadInspectorHelper.h

```diff
@@ -632,4 +632,330 @@ namespace BeamSpotPI {
   };
 }  // namespace BeamSpotPI
 
+// Similar namespace for SimBeamSpotObject
+namespace SimBeamSpotPI {
```

Comment:
Namespace should be lowercase, see 2.7 https://cms-sw.github.io/cms_coding_rules.html